src/term/mod.rs

```diff
@@ -786,3 +789,9 @@ impl Deref for Name {
     self.0.deref()
   }
 }
+
+impl Book {
+  pub fn entrypoint(&self) -> String {
```

Comment:
This is the hvmc entrypoint right? Also it's a bit confusing to call it the same name as one of the fields of `Book`, so let's change it to `Book::hvmc_entrypoint`.